AutoFlow_OmicsDataHandling/INCA_script_generator.py

```diff
@@ -0,0 +1,1232 @@
+                            rxn_equation + reactant_equation + product_equation
+                        )
+        # the ids of the equations are being added to the list that
+        # will be exported
+        model_rxn_ids_exp.append(model_rxn_id)
+        mat_script += rxn_equation + " ';...\n"
+    mat_script += "});\n\n"
+
+    return mat_script, model_rxn_ids_exp
+
+
+def initialize_model():
+    """
+    Previously described reactions are assigned to a model object
+
+    Returns:
+        mat_script: addition to MATLAB script under construction
+
+    """
+    mat_script = "m = model(r); % set up model\n\n"
+
+    return mat_script
+
+
+def symmetrical_metabolites(atomMappingMetabolite_data_I):
+    """
+    Takes care of symmetrical metabolites if not done so in the
+    reaction equations
+
+    Parameters:
+        atomMappingMetabolite_data_I: pre-processed
+            atomMappingMetabolite_data_I input data
+
+    Returns:
+        mat_script: addition to MATLAB script under construction
+
+    """
+    tmp_script = "% take care of symmetrical metabolites\n"
+    for cnt_met, met in atomMappingMetabolite_data_I.iterrows():
+        if not pd.isna(met["met_symmetry_atompositions"]):
+            tmp_script = (
+                tmp_script
+                + "m.mets{'"
+                + met["met_id"]
+                + "'}.sym = list('rotate180', atommap('"
+            )
+            met_positions = [
+                int(x)
+                for x in prepare_input(met["met_atompositions"], "Curly")
+            ]
+            met_elements = prepare_input(met["met_elements"], "Curly")
+            met["met_symmetry_elements"] = met[
+                "met_symmetry_elements"
+            ].replace("{}", str(np.nan))
+            met_symmetry_elements = prepare_input(
+                met["met_symmetry_elements"], "Curly"
+            )
+            met["met_symmetry_atompositions"] = met[
+                "met_symmetry_atompositions"
+            ].replace("{}", str(np.nan))
+            met_symmetry_atompositions = [
+                int(x)
+                for x in prepare_input(
+                    met["met_symmetry_atompositions"], "Curly"
+                )
+            ]
+
+            for cnt, atompositions in enumerate(met_positions):
+                tmp_script = (
+                    tmp_script
+                    + met_elements[cnt]
+                    + str(atompositions + 1)
+                    + ":"
+                    + met_symmetry_elements[cnt]
+                    + str(met_symmetry_atompositions[cnt] + 1)
+                    + " "
+                )
+            tmp_script = tmp_script[:-1]
+            tmp_script = tmp_script + "'));\n"
+        else:
+            continue
+    tmp_script = tmp_script + "\n"
+    mat_script = tmp_script
+
+    return mat_script
+
+
+def unbalanced_reactions(atomMappingMetabolite_data_I):
+    # NOTE: hard-coded for now until a better workaround can be done
```

Comment:
I think this comment will break the docstrings rendering? I would suggest to remove or add to the docstrings itself.